### ISSUE TITLE
file type misassignment

### ISSUE THREAD

**Issue by hadigh** [usgs/groundmotion-processing#799]
**Describe the bug**
I have noticed that in couple of  occasions, the input raw data in miniseed format are detected as "renadic" format! It does not generate any error message during the assembly or process commands as the read command is within "try" block; however, as expected such traces are not actually read by the program, returning empty workspaces.   
 
**To Reproduce**
I have attached a sample data to this message 

**Environmnet (please complete the following information):**
 - OS: Ubuntu 18.04
 - Python Version 3.8.6
- Libcomcat Version 2.0.13
- Numpy Version 1.21.4
[19900508184056.zip](https://github.com/usgs/groundmotion-processing/files/7622915/19900508184056.zip)

**Comment by smithj382** [comment]
@hadigh 

I took a look at the sample data and confirmed that obspy is able to read it as a miniseed format.

```
from obspy import read
tr=read(filename)
print(tr[0].stats)
```

In order to debug this, I used the logger with the debug option (```gmprocess -l -d assemble -o```).  But this was not able to provide much insight.

```
DEBUG 2021-12-01 10:30:51 | read_directory.directory_to_streams: Attempting to read: /Users/jamesasmith/.gmprocess/../gmprocess_projects/TestFile/data/19900508184056/tmpgzq4walk/directory_to_streams/AU.DOW..HNZ__20110823T175034Z__20110823T175804Z.mseed
DEBUG 2021-12-01 10:30:51 | core.is_obspy: Checking if format is supported by obspy.
YES
DEBUG 2021-12-01 10:30:51 | core.is_esm: Checking if format is esm.
DEBUG 2021-12-01 10:30:51 | core.is_knet: Checking if format is knet.
DEBUG 2021-12-01 10:30:51 | core.is_geonet: Checking if format is geonet.
DEBUG 2021-12-01 10:30:51 | core.is_cosmos: Checking if format is cosmos.
```

So I tried running the following code:

```
from gmprocess.io.read import read_data
workspace = read_data(filename)
```

and this indicated where the bug is located:

```
~/Packages/groundmotion-processing/gmprocess/io/renadic/core.py in <listcomp>(.0)
     67     """
     68     with open(filename, 'rt', encoding=ENCODING) as f:
---> 69         lines = [next(f) for x in range(TEXT_HDR_ROWS)]
     70     if MARKER in lines[7]:
     71         return True

StopIteration: 
```

So I used added a try/except block (a bit of a hack) to fix this error:

```
    with open(filename, 'rt', encoding=ENCODING) as f:
        try:
            lines = [next(f) for x in range(TEXT_HDR_ROWS)]
        except:
            return False
    if MARKER in lines[7]:
        return True
```

This fixed the issue, but I would like some feedback form @emthompson-usgs  and other experienced coders for a better way to patch this. 

I'll mention that the mseed format is detected and read in binary by obspy so that issues of wrong encodings and reading in invalide unicode may be less of an issue. But trying to implement a binary reader to detect the format is not simple.

**Comment by emthompson-usgs** [comment]
I'm out of the office and will not be able to look into this for a while. But also @mhearne-usgs wrote this reader initially. Mike: Can you look into this? If I recall correctly, this was not an especially well documented format, and having a reliable reader may be impossible. Or maybe having more example data, as provided by @hadigh here, will help improve the reader.

**Comment by emthompson-usgs** [comment]
Closed by #806. Thanks @mhearne-usgs.

**Comment by hadigh** [comment]
Thanks
